com.github.matfantinel.reminduck.json

```diff
@@ -0,0 +1,87 @@
+{
+    "app-id": "com.github.matfantinel.reminduck",
+    "runtime": "org.gnome.Platform",
```

Comment:
Any reason for not using the freedesktop runtime here?

Reply:
I have based my manifest on another app, and don't really know the difference between the two

Reply:
All the runtimes/sdk's are based on freedesktop runtime/sdk. The GNOME runtime/sdk contains libraries and tools specific to GNOME like Vala. As you're already using the elementary baseApp which contains vala + granite & the elementary stylesheet & icons, you can just switch to the freedesktop runtime.